docs/model-monitoring/index.md

```diff
@@ -49,13 +49,21 @@ For example:
 ## Selecting the streaming and TSDB platforms
 
 Model monitoring supports Kafka or V3io as streaming platforms, and TDEngine or V3IO as TSDB platforms.
-In addition, internal model-monitoring metadata can be saved in MySQL (8.0 and higher) or V3IO.  
+In addition, internal model-monitoring metadata can be saved in MySQL or V3IO.
+
+We recommend the following versions:
+* TDEngine - `3.3.2.0`.
+* MySQL - `8.0.39`, or higher `8.0` compatible versions.
+* Kafka - `3.x` compatible versions, although older version may also work.
```

Comment:
For now please remove the sentence about Kafka, I don't think we currently have "formal" spec of what kafka versions are supported across mlrun, nuclio etc (I'll take it with Orit)

Reply:
Okay, I will remove it. Just take into account that the oldest actively supported version is `3.3`:
https://endoflife.date/apache-kafka